Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Description: Debian doesn't ship or encourage bundled/vendored mechanisms

Furthermore, these additional variants create a huge Provides: line in
debian/control, which itself can cause problems for some tooling.

It would be preferable to declaratively omit these features, but until
https://bugs.debian.org/1014613 is closed in debcargo, we can just
patch them.

We also drop in_gecko, since this is not how libsqlite3-sys is built
on debian.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -151,19 +151,2 @@
 buildtime_bindgen = ["libsqlite3-sys/buildtime_bindgen"]
-bundled = [
-    "libsqlite3-sys/bundled",
-    "modern_sqlite",
-]
-bundled-full = [
-    "modern-full",
-    "bundled",
-]
-bundled-sqlcipher = [
-    "libsqlite3-sys/bundled-sqlcipher",
-    "bundled",
-]
-bundled-sqlcipher-vendored-openssl = [
-    "libsqlite3-sys/bundled-sqlcipher-vendored-openssl",
-    "bundled-sqlcipher",
-]
-bundled-windows = ["libsqlite3-sys/bundled-windows"]
 collation = []
@@ -178,6 +161,2 @@
 i128_blob = []
-in_gecko = [
-    "modern_sqlite",
-    "libsqlite3-sys/in_gecko",
-]
 limits = []
@@ -210,1 +189,1 @@
-modern_sqlite = ["libsqlite3-sys/bundled_bindings"]
+modern_sqlite = ["libsqlite3-sys/buildtime_bindgen"]
